packages/manager-components/src/components/card/card.stories.tsx

```diff
@@ -0,0 +1,87 @@
+import { Meta, StoryFn } from '@storybook/react';
+import { ODS_THEME_COLOR_INTENT } from '@ovhcloud/ods-common-theming';
+/* eslint-disable import/order */
```

Comment:
**nitpick:** Dont disable the ESLint rules.

Reply:
👍